test/test.cc

```diff
@@ -251,6 +251,7 @@ std::vector< testsweeper::routines_t > routines = {
     { "gelqf",              test_gelqf,     Section::qr }, // tested numerically
     { "geqlf",              test_geqlf,     Section::qr }, // tested numerically
     { "gerqf",              test_gerqf,     Section::qr }, // tested numerically; R, Q are full sizeof(A), could be smaller
+    { "gemqrt",             test_gemqrt,    Section::qr }, // tested numerically
```

Comment:
Change to "tested via LAPACKE".
Sorry the comments here are unclear. "tested numerically" means it has a backwards error check, while "tested via LAPACKE" means the result is compared to LAPACKE (a forward error check).

Reply:
Fixed.